project1/models.py

```diff
@@ -1,7 +1,15 @@
+'''
+this file is for models in our database
+'''
 from flask_sqlalchemy import SQLAlchemy
 from datetime import datetime as dt
+
+#create a db instance
 db = SQLAlchemy()
 
+'''
+Class for storing user details
+'''
 class User(db.Model):
     __tablename__ = "users"
     id = db.Column(db.Integer,
```

Comment:
We can directly use email as primary key instead of adding another variable which is not often accessed by us.

Reply:
Yes, but we need some index to show up the list of users on the admin side. I make use of id column to that.

Reply:
In that case, we can use an iterator to increment and show the value while rendering the webpage rather than storing them on the database